sys/linux/vmw_vmci.txt.const

```diff
@@ -0,0 +1,46 @@
+# Code generated by syz-sysgen. DO NOT EDIT.
+arches = 386, amd64, arm, arm64, riscv64, s390x
+AT_FDCWD = ???
+IOCTL_VMCI_CTX_ADD_NOTIFICATION = 1967
+IOCTL_VMCI_CTX_GET_CPT_STATE = 1969
+IOCTL_VMCI_CTX_REMOVE_NOTIFICATION = 1968
+IOCTL_VMCI_CTX_SET_CPT_STATE = 1970
+IOCTL_VMCI_DATAGRAM_RECEIVE = 1964
+IOCTL_VMCI_DATAGRAM_SEND = 1963
+IOCTL_VMCI_GET_CONTEXT_ID = 1971
+IOCTL_VMCI_INIT_CONTEXT = 1952
+IOCTL_VMCI_NOTIFICATIONS_RECEIVE = 1958
+IOCTL_VMCI_NOTIFY_RESOURCE = 1957
+IOCTL_VMCI_QUEUEPAIR_ALLOC = 1960
+IOCTL_VMCI_QUEUEPAIR_DETACH = 1962
+IOCTL_VMCI_QUEUEPAIR_SETPAGEFILE = 1961
+IOCTL_VMCI_QUEUEPAIR_SETVA = 1956
+IOCTL_VMCI_SET_NOTIFY = 1995
+IOCTL_VMCI_VERSION = 1951
+IOCTL_VMCI_VERSION2 = 1959
+O_RDWR = ???
```

Comment:
This is defined by some common header file, you can find it in lots of other description files. Please include it here as well.